configure.ac

```diff
@@ -1438,9 +1438,12 @@ if test "$use_external_signer" != "no"; then
     ;;
     *)
       AC_MSG_CHECKING([whether Boost.Process can be used])
+      TEMP_LDFLAGS="$LDFLAGS"
+      LDFLAGS="$TEMP_LDFLAGS $PTHREAD_CFLAGS"
```

Comment:
Maybe document why this is necessary and when it can be droped, e.g.:
```
dnl This check requires the -lpthread flag.
del This workaround can be dropped when the minimum Boost is 1.78 (possibly earlier)
```

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/24397#issuecomment-1047679471).